habushu-maven-plugin/src/test/resources/junit-platform.properties

```diff
@@ -1,2 +1,6 @@
 cucumber.publish.quiet=true
-cucumber.publish.enabled=false
\ No newline at end of file
+cucumber.publish.enabled=false
+cucumber.glue=org.technologybrewery.habushu
+cucumber.filter.tags=~@manual
```

Comment:
A: I may be able to define this as "not manual" and remove one of the changes above.